index.html

```diff
@@ -228,45 +226,77 @@
   <dd>This attribute MUST return a <a href="http://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp">DOMHighResTimeStamp</a> equal to the difference between <a href="#widl-PerformanceNavigationTiming-loadEventEnd">loadEventEnd</a> and <a href="#widl-PerformanceNavigationTiming-startTime">startTime</a>, respectively.</dd>
 </dl>
 
-<dl title='interface PerformanceNavigationTiming : PerformanceEntry' class='idl'>
+</section>
 
-<dt>readonly attribute DOMHighResTimeStamp unloadEventStart</dt>
-<dd>
-<p>If the previous document and the current document have the same
-<a href="http://tools.ietf.org/html/rfc6454">origin</a>
-[[!RFC6454]],
-this attribute MUST return a <a href="http://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp">DOMHighResTimeStamp</a>
-with a time value equal to the time immediately before the user agent starts
-the <a href="http://www.w3.org/TR/html5/browsers.html#unloading-documents">unload event</a> of the previous document.
+<section id='PerformanceResourceTiming'>
+<h3>Relation with the <a href="http://www.w3.org/TR/resource-timing/#idl-def-PerformanceResourceTiming">PerformanceResourceTiming</a> interface</h3>
 
-If there is no previous document or the previous document has a
-different <a href="http://tools.ietf.org/html/rfc6454">origin</a> than the
-current document, this attribute MUST return a <a href="http://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp">DOMHighResTimeStamp</a>
-with a time value equal to zero.</p>
-</dd>
+<p>Only the <a>current document</a> resource gets included as the <em>only</em> <a>PerformanceNavigationTiming</a> object in the Performance Timeline of the relevant context.</p>
 
-<dt>readonly attribute DOMHighResTimeStamp unloadEventEnd</dt>
+<p>The <a>PerformanceNavigationTiming</a> interface extends the following attributes of the <a href="http://www.w3.org/TR/resource-timing/#idl-def-PerformanceResourceTiming">PerformanceResourceTiming</a> interface:</p>
+
+<dl class='attributes'>
+  <dt id='widl-PerformanceNavigationTiming-initiatorType'><code>initiatorType</code></dt>
+  <dd>This attribute MUST return the empty <code>DOMString</code> "".</dd>

```

Comment:
Maybe the word extend isn't the best here. My goal was to return the empty string "" because initiatorType doesn't really make sense and we already have "NavigationType type". An alternative is to have initiatorType return the same value as type.